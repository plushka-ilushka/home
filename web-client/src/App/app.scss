@import "../assets/colors.scss";
@import "../assets/fonts.scss";

body {
  padding: 0;
  margin: 0;
  font-family: $FONT_FAMILY;
  width: 100%;
  height: 100%;
}

p,
h1 {
  padding: 0;
  margin: 0;
}

h1 {
  padding: 20px 0 20px 0;
}
