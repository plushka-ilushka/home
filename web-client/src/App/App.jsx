import React from 'react';

import Icon from '../common/components/icon';

import shape from '../assets/icons/shape.svg';
import key from '../assets/icons/key.svg';
import suche from '../assets/icons/suche.svg';

import './app.scss';

export default function App() {
  return (
    <div>
      <Icon glyph={shape.id} viewBox={shape.viewBox} />
      <Icon glyph={key.id} viewBox={key.viewBox} />
      <Icon glyph={suche.id} viewBox={suche.viewBox} />
    </div>
  );
}
