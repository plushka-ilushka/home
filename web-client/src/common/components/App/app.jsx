import React, { Component } from 'react';

import Input from '../input';
import Button from '../button';
import Icon from '../icon';
import Image from '../image';

import './app.scss';

export default class App extends Component {

  render() {
    return (
      <div>
        <Input className="input" name="Email" label="Email" htmlFor="Email" title="Email" />
        <Input className="input" type="password" name="password" label="password" htmlFor="password" title="Password" />
        <Input className="check-box-outline-blank" type="checkbox" name="checkbox" label="checkbox" htmlFor="checkbox" title="Remember me" />
        <Button className="blue-outline button" value="Register" />
        <Button className="base button" value="Login" />
        <Icon />
        <Image />
      </div>
    );
  }
}
