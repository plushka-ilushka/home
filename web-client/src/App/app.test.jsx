import React from 'react';
import { mount } from 'enzyme';
import App from './app';

describe('App Component', () => {
  it('check App input', () => {
    expect(mount(<App />)).toMatchSnapshot();
  });
});
