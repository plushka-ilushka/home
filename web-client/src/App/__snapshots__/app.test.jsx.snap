// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component check App input 1`] = `
<App>
  <div>
    <Image
      alt="f"
      src="f"
    >
      <img
        alt="f"
        src="f"
      />
    </Image>
  </div>
</App>
`;
